src/util/VRControllerManager.js

```diff
@@ -128,7 +128,7 @@ x3dom.VRControllerManager.prototype.fit = function ( viewarea, vrDisplay )
 
     var aspect =  Math.min( viewarea._width / viewarea._height, 1 );
 
-    var tanfov2 = Math.tan( 1.57 / 2.0 );
+    var tanfov2 = Math.tan( Math.PI / 4.0 );
```

Comment:
I believe tanfov2 means tangents of field of view divided by 2.
`Math.tan( 0.5 * Math.PI / 2.0` would be closer to that.